src/pyorcidator/helper.py

```diff
@@ -137,16 +164,18 @@ def process_item(
     ref,
     qualifier_nested_dictionary=None,
 ):
+    # FIXME this is unused, delete?
```

Comment:
Can we delete this function? It's unused (and also very hard to understand)

Reply:
I agree with deleting it, don't fully understand it either. Any comments @lubianat?